test/e2e/testkit/matchers/metric_matchers_test.go

```diff
@@ -209,6 +209,87 @@ var _ = Describe("HaveSumMetrics", Label("metrics"), func() {
 	})
 })
 
+var _ = Describe("HaveMetricsThatSatisfy", Label("metrics"), func() {
+	var fileBytes []byte
+	//var expectedMetrics []pmetric.Metric
+
+	BeforeEach(func() {
+	})
+
+	Context("with nil input", func() {
+		It("should error", func() {
+			success, err := HaveMetricsThatSatisfy(func(metric pmetric.Metric) bool {
+				return true
+			}).Match(nil)
+			Expect(err).Should(HaveOccurred())
```

Comment:
My code-style preference is to separate the Arrange, Act and Assert blocks of the test with empty lines. IMO it improves the readability, but it is a subjective preference.

Reply:
As discussed, replaced it with a one-liner